### PR TITLE
run CI tests in the CMSSMCPV

### PR DESCRIPTION
**Pull request by wkotlarski**
The $a_\mu$ in this model comes out complex. Something that might help debug the source of the problem: the A2L is real. The A2R is complex.

### CODE REVIEW

**Comment by Expander** [comment]
Can you tack down which vertex/prefactor comes out complex? Maybe we can then find out if this makes sense or not. If it makes sense, then we should just take the real part of the prefactor and do not print a warning/error. If not, we can this information find the bug. :)

**Comment by dhjjacob** [comment]
Hi @wkotlarski, @Expander, I think I've found were the issue is coming from.  On lines 1003-1004 of FFV_form_factors.cpp.in, the definition of mV and mG are reversed.  So basically one of the diagram contributions was using the mass of the vector boson rather than the scalar boson in the denominator.  I've fixed this up and the NaN result has gone away now.

**Comment by wkotlarski** [comment]
Thanks @dhjjacob. I appreciate you looking into this. I know you're busy...

File: templates/amm.cpp.in [hunk 1]
```diff
@@ -314,9 +314,20 @@ double calculate_amm_impl(const @ModelName@_mass_eigenstates& model, const softs
 
 @AMMCalculation@
 
-   if (!is_zero((form_factors[2] + form_factors[3]).imag())) {
-      ERROR("Error in the g-2 calculation! Form factor F2 should be real");
-      return std::numeric_limits<double>::quiet_NaN();
+   // Im[F2]/Re[F2] ~ 0
```
Comment (Expander) [inline templates/amm.cpp.in:317]: I'd be in favor of removing this error handling altogether, if you don't mind.

**Comment by Expander** [comment]
> Hi @wkotlarski, @Expander, I think I've found were the issue is coming from. On lines 1003-1004 of FFV_form_factors.cpp.in, the definition of mV and mG are reversed. So basically one of the diagram contributions was using the mass of the vector boson rather than the scalar boson in the denominator. I've fixed this up and the NaN result has gone away now.

Excellent, thanks a lot for fixing this!

File: templates/amm.cpp.in (hunk 1 above)
Comment (wkotlarski) [reply templates/amm.cpp.in:317]: I'm against removing removing this. This is exactly the thing that let us find out that there's an error in the first place.

Comment (Expander) [reply templates/amm.cpp.in:317]: I see your point, it finds a bug on the FS side. I guess the error message "Error in the g-2 calculation! Form factor F2 should be real" is clear enough to express this.

**Review by Expander — APPROVED** [review]

File: templates/amm.cpp.in (hunk 1 above)
Comment (wkotlarski) [reply templates/amm.cpp.in:317]: Thanks. This is a relatively not costly check. It shouldn't be a major bottleneck.

**Comment by wkotlarski** [comment]
I've fixed the tests. One they pass I'll merge it.

**Pull request merged**